index/org.go

```diff
@@ -288,3 +288,16 @@ func (index *OrgIndex) GetCA(name string) (string, error) {
 	// TODO - check existence
 	return index.Data.Body.CAs[name], nil
 }
+
+func (index *OrgIndex) GetCAs() (map[string]string, error) {
```

Comment:
Why does this have an error as tge second return type but never actually returns an error?